### PR TITLE
Chapter 5 challenge 3 scripting challenge

### LINKED ISSUE #718: Chapter 5 Challenge-3 Part-1 (Vanderpoole message)
Impllement the message creation challenge

image

[Figma](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=5466%3A3776&mode=dev)

### PR DESCRIPTION
**Pull request by 0tuedon**
This Pr closes #718 
Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

**Comment by benalleng** [comment]
You'll need to correct the editable range so that I can test

**Comment by 0tuedon** [comment]
> You'll need to correct the editable range so that I can test

that's weird they work for me on the preview. 
https://saving-satoshi-5p94vz2gr-savingsatoshi.vercel.app/en/chapters/chapter-5/validate-signature-1?dev=true

**Comment by benalleng** [comment]
I am unable to call the function since the last line isn't editable

File: content/lessons/chapter-5/validate-signature-1.tsx [hunk 1]
```diff
@@ -1,40 +1,138 @@
...
+text = "I am Vanderpoole and I have control of the private key Satoshi\\n" 
+text += "used to sign the first-ever Bitcoin transaction confirmed in block #170.\\n" 
+text += "This message is signed with the same private key." 
 
-  const isMediumScreen = useMediaQuery({ width: 1024 })
+def encode_message(text): 
+  # Given an ascii-encoded text message, serialize a byte array 
+  # with the Bitcoin protocol prefix string followed by the text 
+  # and both components preceded by a length byte. 
+  # Returns a 32-byte integer.
```
Comment (benalleng) [inline content/lessons/chapter-5/validate-signature-1.tsx:73]: Still need to add the prefix here in python

**Review by benalleng — CHANGES_REQUESTED** [review]
! small thing with the default code in python.

File: content/lessons/chapter-5/validate-signature-1.tsx [hunk 2]
```diff
@@ -1,40 +1,138 @@
...
+`,
+  validate: async (answer: string) => {
+    if (answer) {
+      if (answer.length === 77) {
+        return [true, 'Nicely Done ']
+      }
+      return [false, 'Not a valid 32 byte integer']
     }
-  }, [isMediumScreen])
+    return [false, '']
```
Comment (benalleng) [inline content/lessons/chapter-5/validate-signature-1.tsx:83]: Also, the validation isn't quite right, It needs to equal exactly `52301120241471554888909249431012931756072043317860288098683158858465117082178` or else little and big endianness both are the correct answer

Comment (0tuedon) [reply content/lessons/chapter-5/validate-signature-1.tsx:83]: alright.

File: content/lessons/chapter-5/validate-signature-1.tsx (hunk 1 above)
Comment (0tuedon) [reply content/lessons/chapter-5/validate-signature-1.tsx:73]: what prefix?

**Review by benalleng — APPROVED** [review]
tACK LGTM!

**Pull request merged**